### DISCUSSION TITLE
Kutular içindeki para sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#274] [Zeka Sorusu] [answered]
A ve B kutularının birinin içinde 10 milyar liralık çek bulunmaktadır.
A kutusunun üzerinde “B’nin üzerindeki yazı kısmen yanlıştır ve çek B’nin içindedir” yazmaktadır.
B kutusunun üzerinde ise “A’nın üzerindeki yazı kısmen doğrudur ve çek B’nin içindedir” yazmaktadır.
Buna göre çek hangi kutudadır?

**Comment by YagizEfeZ** [comment, marked as answer]
Çek B kutusundadır çünkü: B kutusunun üzerindeki yazı "A nın üzerindeki yazı kısmen doğrudur ve çek B nin içindedir" ifadesine uyar çünkü çek gerçekten B kutusunun içindedir Ayrıca A kutusunun üzerindeki yazı da "B nin üzerindeki yazı kısmen yanlıştır ve çek B nin içindedir" ifadesine uyar, ama bu durum geçerli değil çünkü çek B kutusunun içindedir

yani çek B kutusunun içindedir,
